negy-gateway/src/main.rs

```diff
@@ -46,6 +48,7 @@ async fn fetch_nodes_unselected(node_pool_endpoint: &str) -> Result<Vec<NodeUnse
         .await?
         .json::<ListNodeResponse>()
         .await?;
+    let args = Args::parse();
```

Comment:
Avoid `Args::parse` again but pass the min_version as args of `spawn_inner`. It's already parsed in main function.

Reply:
Okay, it now passes args from the main function.